interface/weno/ajax_sample_code.php

```diff
@@ -0,0 +1,124 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+<title>Weno Sample Code</title>
+<link rel="stylesheet" href="../themes/style_purple.css?v=15" type="text/css">
```

Comment:
I fixed this and made it a part of the troubleshooting process 
http://open-emr.org/wiki/index.php/OpenEMR_ePrescribe#Troubleshooting